traits/simplerollouttrait/README.md

```diff
@@ -15,6 +15,9 @@ Please follow [addon-oam-kubernetes-local](https://github.com/crossplane/addon-o
 - Crossplane v0.11+ installed
 - OAM controllers installed
 
+:exclamation: Please notice that, since oam-core-controller enabled new versioning mechanism has not been released yet, you have to setup it with latest code of oam-kubernetes-runtime locally instead of installing it by [offical Helm charts](https://github.com/crossplane/addon-oam-kubernetes-local/tree/master/charts/oam-core-resources). More details refer to [Development.md](https://github.com/crossplane/oam-kubernetes-runtime)
```

Comment:
Component versioning mechanism implementation has been completed by @wonderflow via https://github.com/crossplane/oam-kubernetes-runtime/pull/35. Besides, I think `oam-core-controller` means `oam-kubernetes-runtime`, so you might need to modify the lines.

Reply:
Yes, I will correct it. As for implementation not released, it's merged indeed but I think it hasn't been included in the image used in that helm chart... so users may not enable the mechanism if they use the guidance.